### PR TITLE
NEW: Add pydra pipeline for t1-freesurfer

### PR DESCRIPTION
**Pull request by ghisvail**

### CODE REVIEW

**Comment by ghisvail** [comment]
Requires a few iterations to [pydra-freesurfer](https://github.com/aramis-lab/pydra-freesurfer)

**Comment by ghisvail** [comment]
Requires a new release of `pydra-bids` to fix this annoying error:

```
Traceback (most recent call last):
...
    parsed = parse_bids_name(os.fspath(file_path))
TypeError: expected str, bytes or os.PathLike object, not LazyField
```

**Comment by ghisvail** [comment]
Fixed in `pydra-bids` version 0.0.10

File: clinica/pydra/t1_freesurfer/pipeline.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        ]
+    )
+
+    return workflow
+
+
+def build_workflow(
+    bids_dir: os.PathLike,
+    caps_dir: os.PathLike,
+    atlas_path: os.PathLike,
```
Comment (NicolasGensollen) [inline clinica/pydra/t1_freesurfer/pipeline.py:86]: Did you implement the custom atlas functionality as well ? It looks like `atlas_path` is unused.

**Review by NicolasGensollen — APPROVED** [review]
LGTM ! Thanks @ghisvail !

**Comment by NicolasGensollen** [comment]
Merging this. We'll work on the `atlas_path` feature in future PRs.

**Pull request merged**